src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -32,7 +41,10 @@ const StyledTextArea = styled.textarea<{ height?: string }>`
   resize: none;
   min-height: 40px;
   font-size: 16px;
-
+  ${unica("s16")};
```

Comment:
Is there any way to replace this with the textarea from palette? https://palette.artsy.net/elements/inputs/textarea/

Reply:
yes of course, thanks for the feedback

Reply:
Edit*  **TextArea** `onChange` inside **Palette** is expecting a value and returns void. I am not sure if it is a good idea to change it to support the behavior [here](https://github.com/artsy/force/blob/aa1866dbbeeeacee0251fc942f380ff6a366fcd0/src/v2/Apps/Conversation/Components/Reply.tsx#L127-L138) now. Although I believe it should be able to support it as well, but we'll have to adjust it everywhere where it is used (I would gladly adjust it)

Reply:
Second @dzucconi - we should be using Palette for things like this . If something is missing from the API we should consider changing one side contract and / or updating Palettte with a new prop (if totally necessary).

Reply:
Hm, yeah the way it clobbers the native event is weird/unfortunate.

One option is just to use the input then set as `as="textarea"`

![](http://static.damonzucconi.com/_capture/jh1MhjEYrpj0.png)